tools/xcodeproj_shims/output-processor.rb

```diff
@@ -37,21 +37,21 @@ def self.create_from(line)
 end
 
 # Documentation
-class SnapshotLine < RegexMatchLine
+class StarlarkLine < RegexMatchLine
   def initialize(line)
-    @regex = /^DEBUG: .+ Snapshot test (.+) (\(.+\)) does not specify it uses an app host$/
+    @regex = /^DEBUG: (.+.bzl)(:\d+:\d+:) (.+)$/
 
     super(line)
 
     return unless (@match_data = line.match(@regex))
 
-    test_name, test_path = @match_data.captures
-    @text = "warning: Snapshot test #{test_name} #{test_path} does not specify it uses an app host"
+    starlark_file, file_line, message = @match_data.captures
```

Comment:
should we be pointing to the file with the issue here?

Reply:
We could include offending file path - but in my testing Xcode seems to pick up any output matching `<path>:<line>:<character> ` and assumes it's an error unless `info:` or `warning:` is specified

Reply:
but i think we are specifying `warning` ?

Reply:
Yup, I ended up appending the path to the .bzl file for developers to dig further into if needed... What I was trying to get at was if we do this:

`warning: Some warning message (/some/path/to/a/file:20:10) and some additional warning data`

Xcode **still** flags that an error (as I observed), as lines formatted like that would show up as errors highlighted in red in the build log and in the issue navigator. Removing the line and character marker seems to correct this.

Reply:
I think this had to do with the STDOUT / STDERR though. I couldn't get repeatable behaviour before I made the the change to only route it to STDOUT.